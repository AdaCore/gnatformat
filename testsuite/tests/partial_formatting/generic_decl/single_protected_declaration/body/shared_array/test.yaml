description: ProtectedBOdy formatting from line selection.
driver: partial_gnatformat
args:
  [
    "test.adb",
    "-SL 67",
    "-EL 78",
  ]
status_code: 1
