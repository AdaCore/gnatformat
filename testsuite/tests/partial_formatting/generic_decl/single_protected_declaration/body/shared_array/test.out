<CompilationUnit test.adb:32:1-84:27>
test.adb(<ProtectedBody ["Shared_Array"] test.adb:67:4-77:21>) - 67:1-77:21
^
   protected body Shared_Array is
      function Component (N : in Index) return Item
      is
      begin
         return Table (N);
      end Component;

      procedure Set_Component (N : in Index; E : in Item)
      is
      begin
         Table (N) := E;
      end Set_Component;
   end Shared_Array;$

